Contribute.md

```diff
@@ -0,0 +1,35 @@
+# Dev Install
```

Comment:
Looks like standard filename is contributing.md. https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors